raspyrfm_client/device/base.py

```diff
@@ -35,7 +51,8 @@ def set_channel_config(self, **channel_arguments) -> None:
 
         :param channel_arguments:
         """
-        raise NotImplementedError
+        if self.check_channel_config(**channel_arguments):
+            self._channel = channel_arguments
```

Comment:
May be this way to be more consistent to the original "style":

- have an abstract method get_supported_actions which has to be implemented in the deviceclass (is already in this way)
- have an abstract method get_pulse_data which has to be implemented in the deviceclass
- have an abstract method get_channel_config_args, which has to be implemented in the deviceclass and return e. g. {'ID': '...', 'CH':, '...'}. This would replace method check_channel_config. OR: In base class check_channel_config return True, can be implemented in device classes in order to have more advanced checks